frontend/src/router.js

```diff
@@ -40,6 +41,7 @@ const router = (
               <Route exact path="/admin" component={Admin} />
               <Route exact path="/login" component={Login} />
               <Route exact path="/resources" component={Resources} />
+              <Route exact path="/volunteer" component={Volunteer} />
```

Comment:
lets update the current Volunteer link in the footer to use this new route

Reply:
?

Reply:
update this link here https://github.com/the-difference-engine/ymim/blob/staging/frontend/src/components/common/footer/index.jsx#L42 to point to `/volunteer`